test/commons/text/unicode.js

```diff
@@ -69,6 +69,13 @@ describe('text.hasUnicode', function() {
 			});
 			assert.isTrue(actual);
 		});
+
+		it('returns true for a srting with characters in the Supplementary Private Use Area-A', function() {
+			var actual = axe.commons.text.hasUnicode('\uDB80\uDC19', {
```

Comment:
The string here is the same as the unicode escape sequence `'\u{F0019}'`. However, ES5 doesn't support those, and the tests are parsed by eslint as ES5. Instead this is the corresponding surrogate pair.